src/app/services/ccd-config/ccd-case.config.spec.ts

```diff
@@ -2,6 +2,7 @@ import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { inject, TestBed } from '@angular/core/testing';
 import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { StoreModule } from '@ngrx/store';
+import { EnvironmentConfig } from 'api/interfaces/environment.config';
```

Comment:
should not be using api interfaces if needed on src, should replicate EnvironmentConfig in src